src/middleware.ts

```diff
@@ -35,6 +41,23 @@ const getScoreBoard = async (listType: string, scoreType: string) => {
         if (x.score !== 0) return x;
         return undefined;
     }).filter((y) => y);
+
+    if(enableLevel) {
+        const levelScoreList = scoreList.map(x => {
+            let score = x.score;
+            const threshold: number = scoreRotation;
```

Comment:
Can move this out of the map to avoid recreating it on each iteration. Also, can just use `scoreRotation`, no need to assign it to a new variable.